documentation/asciidoc/computers/configuration/uart.adoc

```diff
@@ -96,6 +122,10 @@ The following table summarises the assignment of the first two UARTs:
 | Raspberry Pi 4
 | secondary (Bluetooth)
 | primary
+
```

Comment:
I would take Pi 5 out of this table - it's Bluetooth is controlled by yet another UART, not previously discussed. It can't be used for anything else, and with the latest kernels it doesn't get a `/dev/tty*` entry, so it's almost as if it doesn't exist.

Alternatively, change the table column headings to be Model, Primary, Secondary and put the UART numbers in each row (with a special note for Pi 5).

Reply:
Change the column headings to "Primary/console" and "Secondary/Bluetooth". Drop the third column - it's all not needed (and mostly wrong).